packages/roboto-1.1.0.yaml

```diff
@@ -0,0 +1,5 @@
+name: roboto
+description: Roboto sans-serif fonts for Toit devices.
+license: apache-2.0
```

Comment:
Currently, casing must follow the spdx list. (there are http-get calls that can find the license. but only if the casing is correct).
```suggestion
license: Apache-2.0
```